frontend/packages/pipelines-plugin/src/components/pipelines/const.ts

```diff
@@ -46,3 +48,17 @@ export const SecretAnnotationType = {
 
 export const PIPELINE_SERVICE_ACCOUNT = 'pipeline';
 export const PIPELINE_RUN_AUTO_START_FAILED = `bridge/pipeline-run-auto-start-failed`;
+
+export const PipelineMetricsTimeRangeOptions = (t: TFunction) => ({
```

Comment:
This is a more of a util than a const, should we position it here?

Reply:
or could we simply move the usage of t: TFunction at a util level?

Reply:
Sure, will move it.